From a02355bdc8cde1115daf101675cdfb707d337bf1 Mon Sep 17 00:00:00 2001
From: Joshua Watt <JPEWhacker@gmail.com>
Date: Mon, 17 Jun 2019 10:47:23 -0500
Subject: [PATCH] Constant: Fix up shebang

The instructions indicate that the script should be explicitly passed to
"perl -x", so automatically setting the #! to be ^X is unnecessary and
makes the file non-reproducible when building because ^X could be the
absolute path to miniperl.

Signed-off-by: Joshua Watt <JPEWhacker@gmail.com>
Upstream-Status: Submitted [https://rt.cpan.org/Public/Bug/Display.html?id=129866]
---
 cpan/ExtUtils-Constant/lib/ExtUtils/Constant/XS.pm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cpan/ExtUtils-Constant/lib/ExtUtils/Constant/XS.pm b/cpan/ExtUtils-Constant/lib/ExtUtils/Constant/XS.pm
index 14eb809..d4d074e 100644
--- a/cpan/ExtUtils-Constant/lib/ExtUtils/Constant/XS.pm
+++ b/cpan/ExtUtils-Constant/lib/ExtUtils/Constant/XS.pm
@@ -219,7 +219,7 @@ sub dogfood {
      Regenerate these constant functions by feeding this entire source file to
      perl -x
 
-#!$^X -w
+#!/usr/bin/env perl -x -w
 use ExtUtils::Constant qw (constant_types C_constant XS_constant);
 
 EOT
